code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -411,12 +411,25 @@
 	// They stay locked down if their wire is cut.
 	if(wires?.is_cut(WIRE_LOCKDOWN))
 		state = TRUE
+	else
+		if(!ai_lockdown)
+			addtimer(CALLBACK(src,PROC_REF(lockdown_override), FALSE), 10 MINUTES)
+			to_chat(src, "<br><br>[span_alert("ALERT - Remote system lockdown engaged. Trying to hack the lockdown subsystem...")]<br>")
 	if(state)
 		throw_alert(ALERT_HACKED, /atom/movable/screen/alert/locked)
 	else
 		clear_alert(ALERT_HACKED)
 	set_lockcharge(state)
 
+/// Allows the borg to unlock themselves after a lenghty period of time.
+/mob/living/silicon/robot/proc/lockdown_override()
+	if(!ai_lockdown)
```

Comment:
```suggestion
	if(ai_lockdown)
		to_chat(src, "<br><br>[span_alert("ALERT - Remote system lockdown override failed.")]<br>")
		return
```

No reason to have most of the proc indented if we can early return with the other, shorter case.